_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,277 @@
+
+| Attribute Type | Attribute Owner | Possible? | Automation      | Examples/Reason                                                                                                                                                                                                                                                       |
+| -------------- | --------------- | --------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          | N         |                 | Use ShareAttributeRequestItem instead.                                                                                                                                                                                                                                |
+| Identity       | Recipient       | Y         | `USER_DECISION` | University sends student his certificate (Propose would be inappropriate in this case, because the student should not be able to return his own value)                                                                                                                |
+| Identity       | `<empty>`       | Y         |                 | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
+| Relationship   | Sender          | Y         | `AUTO_ACCEPT`   | Company sends new customer his customer number.                                                                                                                                                                                                                       |
+| Relationship   | Recipient       | Y         | `USER_DECISION` | With this combination the **sender asks the recipient for the one-time permission** to write a Relationship Attribute once AND the **sender defined a value** which can either be accepted and stored, or rejected. Thus, the user cannot change the value by itself. |
+| Relationship   | `<empty>`       | Y         |                 | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
+
+## ProposeAttributeRequestItem
+
+The ProposeAttributeRequestItem is a combination of ReadAttributeRequestItem and CreateAttributeRequestItem. The sender would like to receive a correct Attribute from the peer, thinks it has a possible value but the peer might overrule this value with an existing or new one.
```

Comment:
Vielleicht solltest du erst das ReadAttributeRequestItem behandeln, weil du hier ja auf dieses verweist.

ACHTUNG: KANNST DU DAS BITTE IN EINEM EIGENEN COMMIT UND ERST GANZ AM ENDE MACHEN, NACHDEM DU ALLE ANDEREN KOMMENTARE BEARBEITET HAST? SONST WIRD DIE DIFF-ANSICHT KOMPLETT VERWIRREND.

Danke. :-)

Reply:
hab die extra alphabetisch sortiert. denke die Verlinkung reicht.